### ISSUE TITLE
Results on YCBInEOAT dataset.

### ISSUE THREAD

**Issue by JiChun-Wang** [wenbowen123/iros20-6d-pose-tracking#39]
Hi, bowen, sorry to bother you again. When I run predit.py  to evaluate the nine video sequences of YCBInEOAT dataset. All sequences except for the 'bleach_hard_00_03_chaitanya' one perform well. I just use your provided pre-trained weights and the translation normalizer and translation normalizer are set to 0.03m and 30 degree respectively. The following are several visulization results:
![0000001](https://user-images.githubusercontent.com/41424692/160276916-e2398a19-9ce3-407f-8176-fea993b067bb.png)
![0000050](https://user-images.githubusercontent.com/41424692/160276942-9f51fd6a-d6bf-4360-bfbc-45e3d5914f81.png)
![0000100](https://user-images.githubusercontent.com/41424692/160277089-2bbfe126-3ccb-424e-899a-e0b2ff858a8d.png)
![0000150](https://user-images.githubusercontent.com/41424692/160276949-ae767fac-d2ac-48ef-94f3-c401b57da633.png)
![0000323](https://user-images.githubusercontent.com/41424692/160276959-d25e2f5c-f022-4210-9d92-992637341ac0.png)
Hope for your assistance.

**Comment by wenbowen123** [comment]
This doesn't look like it's set up correctly. Can you make sure this is set to the correct `dataset_info.yml`?  It should be pointing to some file under the training data you downloaded from https://archive.cs.rutgers.edu/archive/a/2020/pracsys/Bowen/iros2020/YCBInEOAT_weights.zip
https://github.com/wenbowen123/iros20-6d-pose-tracking/blob/1c28f02b93aa282a706181d1b28e9a56a9511d3c/predict.py#L630

**Comment by JiChun-Wang** [comment]
Actually, the other eight video sequences can perform well. I think the necessary setup includes the camera intrinsic, the original image size, the target resolution and the type of renderer and if that's true, it should be set up correctly.

**Comment by wenbowen123** [comment]
I just verified from my side and didn't encounter this problem. From your image it looks like it lost tracking very early in the video. The scene so far in the video is not challenging, so it's still likely there is something wrong. Notice the `dataset_info.yml` here is different from the one used for YCB-Video data.

**Comment by JiChun-Wang** [comment]
Thanks for your reply, I will try again.
